README.md

```diff
@@ -0,0 +1,112 @@
+This is a proxy that adds Twitch authentication to [Pogly](https://github.com/PoglyApp/pogly-standalone), a real-time collaborative stream overlay.
+
+If you want to develop locally, you can follow the instructions [here](#Developing-locally).
```

Comment:
Instead of having this sentence here, can we move the developing locally info to the top first, followed by deployment? I would think environment variables should be after developing locally, as it will be needed for local setup as well, before deploying?